backend/shared/rest/src/main/java/com/meemas/shared/BooleanExpression.java

```diff
@@ -0,0 +1,16 @@
+package com.meemas.shared;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
```

Comment:
```suggestion
@Value
```

Value is an immutable version of `@Data`